workflow.rst

```diff
@@ -160,6 +160,10 @@ follows:
     If you are creating your first workflows, consider using the ``workflow:dump``
     command to :doc:`debug the workflow contents </workflow/dumping-workflows>`.
 
+.. tip::
+
+    Since Symfony 3.2, you can use constants in YAML files and add some in the BlogPost entity for places ``draft`` by ``!php/const App\Entity\BlogPost::STATE_DRAFT`` or for transitions ``to_review`` by ``!php/const App\Entity\BlogPost::TRANSITION_TO_REVIEW``.
```

Comment:
I don't know if it's relevant to indicate that it's since Symfony 3.2, are we used to refer to old versions?

Reply:
Yes there's no ned to add the reference to Symfony 3.2. So, let's remove _"Since Symfony 3.2, "_ and keep the rest. Thanks.